buildGradleApplication/default.nix

```diff
@@ -58,7 +65,7 @@
     done
   '';
 
-  package = stdenvNoCC.mkDerivation {
+  package = lib.makeOverridable stdenvNoCC.mkDerivation ((cleanAttrs attrs) // {
```

Comment:
Isn't this possible by using [`overrideAttrs`](https://nixos.org/manual/nixpkgs/stable/#sec-pkg-overrideAttrs)? If I understand this correctly, [`overrideDerivation` and hence `makeOverridable`](https://nixos.org/manual/nixpkgs/stable/#sec-pkg-overrideDerivation) should be avoided

Reply:
idk what `lib.makeOverridable` even does. I just removed it for now.

Reply:
If you need, I can also get rid of the `((cleanAttrs attrs) //` part (and the cleanAttrs function). It also isn't nessesary to implement this pr. I just added it to try different approaches more easily during testing (e.g. if I really had to hardcode `dontWrapGapps=true` and could just do it in the package definition). I kept it, because I didn't see the harm in allowing people to use the extra functionality.

Reply:
Yeah, I don't get the `cleanAttrs` part 100%, but we can merge `dontWrapGApps = true;` and the adapted `postFixup` phase :+1:

Reply:
Allright. I've removed the `cleanAttrs` as well now.

The `((cleanAttrs attrs) //` just allows you to extend the derivation easier.